catalyst_voices/apps/voices/lib/pages/account/unlock_keychain_dialog.dart

```diff
@@ -157,6 +160,7 @@ class _UnlockPassword extends StatelessWidget {
         errorText: error?.message(context),
         hintText: context.l10n.passwordLabelText,
       ),
+      onSubmitted: (val) => onUnlock.call(),
```

Comment:
```suggestion
      onSubmitted: (val) => onUnlock(),
```

Reply:
I was wondering which way this function should be called.